kiwi/system/root_init.py

```diff
@@ -84,6 +85,8 @@ def create(self):
             data.sync_data(
                 options=['-a', '--ignore-existing']
             )
+            if Defaults.is_buildservice_worker():
+                copy('/.buildenv', self.root_dir)
```

Comment:
This assumes a buildservice worker has to provide /.buildenv which is most probably always true, but if not we will fail here which might be intentional and the correct reaction, however in the legacy kiwi we called

```
touch $root/.buildenv
```

which would never fail.